skin/base-quarantine.css

```diff
@@ -0,0 +1,86 @@
+/**
```

Comment:
Fine for working, but we should not merge it with this file. Note: Please be very careful about removing things - when in doubt, just leave it as there are many different OS's and modes (tabs on top, tabs in reverse, shrunk and expanded) to test in things might surprise you.